sdk/src/test/java/io/radar/sdk/RadarTest.kt

```diff
@@ -1111,6 +1111,60 @@ class RadarTest {
         assertAddressesOk(callbackAddresses)
     }
 
+    @Test
+    fun test_Radar_autocomplete_success_2() {
+        permissionsHelperMock.mockFineLocationPermissionGranted = false
+        apiHelperMock.mockStatus = Radar.RadarStatus.SUCCESS
+        apiHelperMock.mockResponse = RadarTestUtils.jsonObjectFromResource("/search_autocomplete.json")
+
+        val near = Location("RadarSDK")
+        near.latitude = 40.78382
+        near.longitude = -73.97536
+
+        val latch = CountDownLatch(1)
+        var callbackStatus: Radar.RadarStatus? = null
+        var callbackAddresses: Array<RadarAddress>? = null
+
+        Radar.autocomplete("brooklyn roasting", country="UK", expandUnits = true) { status, addresses ->
```

Comment:
Nice testing the named params like this -- why don't we also test with named param for `countryCode` as well